src/ui/inventory.rs

```diff
@@ -109,6 +109,10 @@ pub fn build_menu(api: &mut api::API, machine_status: &Value, machine_index: i32
               ui_common::destroy_win(win);
               return;
             },
+            113 => {
```

Comment:
Are there not constants for these things?

Reply:
there are in a commit i haven't pushed yet.